src/components/products/productsNav.js

```diff
@@ -1,50 +1,53 @@
 import { Link } from "react-router-dom";
 import React, { useState } from "react";
 import classNames from "classnames";
-import "../home/navbar.css";
+import "./productsNav.css";
 export function ProductsNav() {
   const [isSidebarMenu, setIsSidebarMenu] = useState(false);
   const showSidebar = () => {
     setIsSidebarMenu(!isSidebarMenu);
   };
 
   return (
-    <div>
+    <>
+   
```

Comment:
This navigation header is mostly the same as the one for the cart page, hence you can create one global navigation component and place it in App.js